_includes/footer.html

```diff
@@ -1,5 +1,4 @@
 <!-- general behavior -->
-<script type="module" src="{{ site.baseurl }}/scripts/dist/modern/index.js"></script>
-<script nomodule src="{{ site.baseurl }}/scripts/dist/legacy/index.js"></script>
+<script type="module" src="{{ site.baseurl }}/scripts/dist/index.js"></script>
```

Comment:
The `package.json` still uses a named target. I would assume that `npm run build` still outputs the compiled `index.js` file to the `modern` directory, doesn't it? If not, feel free to disregard.

Reply:
Nope, it doesn't - and this was odd to me as well.